docs/advanced/primitive-layers.md

```diff
@@ -58,7 +58,8 @@ be instanced, or use dynamic geometry:
   import {assembleShaders} from 'deck.gl';
   import {GL, Model, Geometry} from 'luma.gl';
 
-  getModel(gl) {
+  _getModel() {
+    const {gl} = this.context;
```

Comment:
Is this change still needed? The layers that subclass the core layers and override `_getModel()` will be broken.